.github/workflows/linux.yml

```diff
@@ -3,223 +3,249 @@ name: linux
-    steps:
-    - uses: actions/checkout@v2
-    - name: Dependencies
-      run: .github/workflows/dependencies/dependencies_nvcc.sh
-    - name: Build & Install
-      run: |
-        mkdir build
-        cd build
-        cmake ..                                         \
-            -DCMAKE_VERBOSE_MAKEFILE=ON                  \
-            -DAMReX_BUILD_TUTORIALS=ON                   \
-            -DENABLE_PARTICLES=ON                        \
-            -DENABLE_CUDA=ON                             \
-            -DCMAKE_C_COMPILER=$(which gcc-4.8)              \
-            -DCMAKE_CXX_COMPILER=$(which g++-4.8)            \
-            -DCMAKE_CUDA_HOST_COMPILER=$(which g++-4.8)      \
-            -DCMAKE_Fortran_COMPILER=$(which gfortran-4.8)   \
-            -DCUDA_ARCH=6.0
-        make -j 2
+  # # Build libamrex and all tutorials with CUDA
+  # tutorials-cuda:
+  #   name: CUDA@9.1.85 GNU@4.8.5 C++11 Release [tutorials]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #   - uses: actions/checkout@v2
+  #   - name: Dependencies
+  #     run: .github/workflows/dependencies/dependencies_nvcc.sh
+  #   - name: Build & Install
+  #     run: |
+  #       mkdir build
+  #       cd build
+  #       cmake ..                                         \
+  #           -DCMAKE_VERBOSE_MAKEFILE=ON                  \
+  #           -DAMReX_BUILD_TUTORIALS=ON                   \
+  #           -DENABLE_PARTICLES=ON                        \
+  #           -DENABLE_CUDA=ON                             \
+  #           -DCMAKE_C_COMPILER=$(which gcc-4.8)              \
+  #           -DCMAKE_CXX_COMPILER=$(which g++-4.8)            \
+  #           -DCMAKE_CUDA_HOST_COMPILER=$(which g++-4.8)      \
+  #           -DCMAKE_Fortran_COMPILER=$(which gfortran-4.8)   \
+  #           -DCUDA_ARCH=6.0
+  #       make -j 2
 
-  tutorials-dpcpp:
-    name: DPCPP@PubBeta GFortran@7.5 C++17 [tutorials]
-    runs-on: ubuntu-latest
+  # tutorials-dpcpp:
+  #   name: DPCPP@PubBeta GFortran@7.5 C++17 [tutorials]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #   - uses: actions/checkout@v2
+  #   - name: Dependencies
+  #     run: .github/workflows/dependencies/dependencies_dpcpp.sh
+  #   - name: Build & Install
+  #     run: |
+  #       set +e
+  #       source /opt/intel/oneapi/setvars.sh
+  #       set -e
+  #       mkdir build
+  #       cd build
+  #       cmake ..                                           \
+  #           -DCMAKE_VERBOSE_MAKEFILE=ON                    \
+  #           -DENABLE_TUTORIALS=ON                          \
+  #           -DENABLE_PARTICLES=ON                          \
+  #           -DENABLE_DPCPP=ON                              \
+  #           -DCMAKE_C_COMPILER=$(which clang)              \
+  #           -DCMAKE_CXX_COMPILER=$(which dpcpp)            \
+  #           -DCMAKE_Fortran_COMPILER=$(which gfortran)
+  #       make -j 2 tutorials
+
+  tutorials-hip:
+    name: HIP ROCm@3.7 GFortran@9.3 C++17 [tutorials]
```

Comment:
I think we should remove `@3.7` from the name.  It's already outdated.  The CI now runs with ROCm 3.8.

Reply:
Done